### PR TITLE
kernel/binary_manager : Change functions that not use return values t…

### PR DESCRIPTION
**Pull request by YiHyunJin**
…o void types

Binary manager always runs even with a request failure.
So we don't need to get return value from request.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [dae148326a34c20c03192128263279d82e6b2cc8] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [dae148326a34c20c03192128263279d82e6b2cc8] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [2e57ff5d468b8882ea7a26638ec63d87758c050c] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [2e57ff5d468b8882ea7a26638ec63d87758c050c] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [175fca0d8df2e39343ee0e3657a22cad02f3520d] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [175fca0d8df2e39343ee0e3657a22cad02f3520d] - Code Rule Check OK.

File: os/kernel/binary_manager/binary_manager_callback.c [hunk 1]
```diff
@@ -174,7 +174,8 @@ int binary_manager_register_statecb(int pid, binmgr_cb_t *cb_info)
 
 send_result:
 	snprintf(q_name, BIN_PRIVMQ_LEN, "%s%d", BINMGR_RESPONSE_MQ_PREFIX, pid);
-	return binary_manager_send_response(q_name, &response_msg, sizeof(binmgr_statecb_response_t));
+	binary_manager_send_response(q_name, &response_msg, sizeof(binmgr_statecb_response_t));
+	return;
```
Comment (sunghan-chang) [inline os/kernel/binary_manager/binary_manager_callback.c]: unnecessary. let's remove.

File: os/kernel/binary_manager/binary_manager_callback.c [hunk 2]
```diff
@@ -309,8 +310,8 @@ int binary_manager_notify_state_changed(int bin_idx, uint8_t state)
 
 	if (fail_count > 0) {
 		bmdbg("Invalid parameter: bin idx %d, state %d\n", bin_idx, state);
-		return BINMGR_OPERATION_FAIL;
+		return;
 	}
 
-	return BINMGR_OK;
+	return;
```
Comment (sunghan-chang) [inline os/kernel/binary_manager/binary_manager_callback.c]: unnecessary

**Comment by seinfra** [comment]
Target : [463cba3d2af608eabfb21c3808c8b79fb4de6595] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [463cba3d2af608eabfb21c3808c8b79fb4de6595] - Code Rule Check (C++) OK.

File: os/kernel/binary_manager/binary_manager_getinfo.c [hunk 3]
```diff
@@ -131,7 +130,7 @@ int binary_manager_get_info_all(int requester_pid)
 
 	if (requester_pid < 0) {
 		bmdbg("Invalid requester pid %d\n", requester_pid);
-		return ERROR;
+		return ;
```
Comment (jeongchanKim) [inline os/kernel/binary_manager/binary_manager_getinfo.c]: Please remove unnecessary space.

**Comment by seinfra** [comment]
Target : [93e24088aaf99f35841ce825984525daed267539] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [93e24088aaf99f35841ce825984525daed267539] - Code Rule Check (C++) OK.

**Pull request merged**